life/modernized/04-chapter-4.adoc

```diff
@@ -37,10 +37,13 @@ Thomas Ellwood, who was present,
 communicated the result of the dispute to one of
 his absent friends in the following laconic epistle:
 
-Prsevaluit Veritas: inimici terga dedere: Nos sumus in tuto: laus tribuenda Deo.
+[quote]
+____
+_Prsevaluit Veritas: inimici terga dedere: Nos sumus in tuto: laus tribuenda Deo._
```

Comment:
So this is stated as an epistle, but I labeled it as quote because that's what I've done in the past with this Latin stuff. Idk if that was right, and the second part looks a little funny.